simulation/traffic_simulator/include/traffic_simulator/entity/entity_manager.hpp

```diff
@@ -542,8 +540,17 @@ class EntityManager
             lanelet_pose) {
           entity_status.lanelet_pose = *lanelet_pose;
           entity_status.lanelet_pose_valid = true;
+          // @note fix z, roll and pitch to fitting to the lanelet
+          static bool consider_pose_by_road_slope =
+            getParameter<bool>("consider_pose_by_road_slope", false);
+          if (consider_pose_by_road_slope) {
+            entity_status.pose = hdmap_utils_ptr_->toMapPose(*lanelet_pose).pose;
+          } else {
+            entity_status.pose = pose;
+          }
```

Comment:
```suggestion
          if (getParameter<bool>("consider_pose_by_road_slope", false)) {
            entity_status.pose = hdmap_utils_ptr_->toMapPose(*lanelet_pose).pose;
          } else {
            entity_status.pose = pose;
          }
```

Reply:
This line is executed when the Entity spawns, so it is not executed every frame.

Reply:
Oh, I see.
Then, it's reasonable.